controllers/helmrelease_controller.go

```diff
@@ -540,7 +540,8 @@ func (r *HelmReleaseReconciler) composeValues(ctx context.Context, hr v2.HelmRel
 				if err := r.Get(ctx, namespacedName, resource); err != nil {
 					if apierrors.IsNotFound(err) {
 						if v.Optional {
-							(logr.FromContext(ctx)).Info("could not find optional %s '%s'", v.Kind, namespacedName)
+							(logr.FromContext(ctx)).
+								Info(fmt.Sprintf("could not find optional %s '%s'", v.Kind, namespacedName))
```

Comment:
Wouldn't these all be better as structured logs, something like:
```suggestion
								Info("could not find optional resource", "kind", v.Kind, "name", namespacedName)
```

Reply:
I was thinking the same thing but I'm not sure about the policy here. @stefanprodan wdyt?

Reply:
If we opt for structured logging, we should ensure that the structures are the same across all GOTK controllers.

This is an OK feature request (and probably would be better), but requires coordination.

Reply:
Makes sense @hiddeco.

Reply:
We do have the reconciled object kind/name/namespace as log labels, I don't think the secret kind/name belongs there as it will overlap with the HelmRelease and users will not be able to tell what's going on.